### PR TITLE
Update pt.json

### PR DESCRIPTION
**Pull request by Tangara-cayana**
- aligned with the em.json file.
- added about 60 lines.
- some syntax changes.
- punctual corrections.
- some terms kept in eng due to being incorporated into the language

### CODE REVIEW

**Comment by xaur** [comment]
I asked in the #translators room if anyone can review this but no response so far.

An alternative approach we can take is "lazy consensus", that is don't block the merge on review and fix translations bugs if anyone reports them. My reasoning is that, a) we don't know when a reviewer for pt-BR shows up (it might take a while), b) when contribs are merged it motivates to do more (and vice versa), c) seeing a translation on the site may actually provoke someone to review or submit more fixes.

**Review by victorgcramos — COMMENTED** [review]
Translation looking okay. Good Job. Left some inline comments.

File: transifex_catalogs/pt.json [hunk 1]
```diff
@@ -1,56 +1,103 @@
...
-  "general_copyright": "Programadores Decred",
-  "nav_brand_resources": "Guia da Marca",
-  "nav_become_contributor": "Contribuindo à Decred",
+  "general_javascript_warning": "JavaScript necessário",
+  "general_javascript_warning_description": "Pedimos desculpas, mas esse site não funciona corretamente sem o JavaScript habilitado.",
+  "exchanges_title": "Corretoras",
+  "exchanges_disclaimer": "Nenhuma das corretoras listadas abaixo, fora a: <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://dex.decred.org/\">DCRDEX</a> empodera os usuários na autocustódia de seus fundos durante o processo de câmbio. Muitas corretoras centralizadas - incluso as \"instatâneas\" - costumam inclusive solicitar dados pessoais para liberar operações após o depósito de seus fundos. Encorajamos fortemente nossos usuários a operarem na DCRDEX para que melhorem a privacidade e segurança de seus investimentos. Este <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.youtube.com/watch?v=XaE_xzGKjeQ\">DCRDEX tutorial</a> está disponível para te ajudar.",
```
Comment (victorgcramos) [inline transifex_catalogs/pt.json:5]:
```suggestion
  "exchanges_disclaimer": "Nenhuma das corretoras listadas abaixo, fora a <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://dex.decred.org/\">DCRDEX</a>, empodera os usuários na autocustódia de seus fundos durante o processo de câmbio. Muitas corretoras centralizadas - incluso as \"instantâneas\" - costumam inclusive solicitar dados pessoais para liberar operações após o depósito de seus fundos. Encorajamos fortemente nossos usuários a operarem na DCRDEX para que melhorem a privacidade e segurança de seus investimentos. Este <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.youtube.com/watch?v=XaE_xzGKjeQ\">tutorial sobre a DCRDEX</a> está disponível para te ajudar.",
```

**Comment by jholdstock** [comment]
@victorgcramos it looks like @Tangara-cayana might not come back. Do you think you could finish this translation and open a fresh PR?

**Comment by dajohi** [comment]
Applied in #1160

**Pull request closed without merging**